bcda/main_test.go

```diff
@@ -264,7 +245,7 @@ func (s *MainTestSuite) TestCreateAlphaTokenCLI() {
 	var outputPattern *regexp.Regexp
 	switch auth.GetProvider().(type) {
 	case auth.AlphaAuthPlugin:
```

Comment:
There shouldn't be any difference in the output patterns between the two plugins now. Please remove the switch.

Reply:
the clientID is not a uuid though i believe

Reply:
in okta that is

Reply:
Both plugins should be return client name, client id, and client secret now. I don't think we should be validating the client id's structure here; that kind of specific testing belongs in the plugin unit tests. We should validate here only that we got outputs for all three fields, and there is nothing unacceptable about those outputs.